msteams-platform/messaging-extensions/what-are-messaging-extensions.md

```diff
@@ -54,6 +129,84 @@ The following image displays the messaging extension search command task module:
 
 ![messaging extension search command](~/assets/images/search-extension.png)
 
+#### Code snippets for search commands
```

Comment:
Make this heading as second level before to the code sample section and remove for search commands from the heading to follow the consistency.

Reply:
Made changes as advised